snap/snapcraft.yaml

```diff
@@ -46,13 +46,3 @@ parts:
         -e 's|Icon=.*|Icon=${SNAP}/usr/share/icons/hicolor/scalable/apps/org.gnome.GTG.svg|'
     after:
       - python-deps
-
-  python-deps:
-    source: https://github.com/getting-things-gnome/liblarch.git
```

Comment:
I have absolutely no experience with snap yet I think you removed here to much. I guess only line 51 (`source: [...]` ) it should be removed here.

Reply:
Maybe @sergiusens would be so kind to check the snapcraft file? He was the last one to contribute to it.

Reply:
Hm yeah it's possible. The other lines kind of look like the dependencies of liblarch, but they may be dependencies of GTG.

I'm not super fussed about this file because I don't think we release snaps regularly, and I don't think we've done it since the GTK4 refactor. So there is a good chance it doesn't work at all.

Happy to go with your suggestion, but if there is further debate I'd like to defer this work and just not touch the file.